mvg/plotting.py

```diff
@@ -210,6 +210,18 @@ def modes_over_time(
     else:
         data["Date"] = pd.to_datetime(data["timestamps"], unit=timeunit)
 
+    # Add an additional row with a future timestamp to ensure the mode
+    # for the last timestamp in the dataset is clearly visible
+    ts_end, ts_prev = data["timestamps"].iloc[[-1, -2]]
+    ts_next = (ts_end - ts_prev) + ts_end
+    data.loc[len(data)] = {
+        "timestamps": int(ts_next),
+        "labels": None,
+        "uncertain": None,
+        "mode_probability": -1,
+        "Date": pd.to_datetime(ts_end, unit=timeunit),
```

Comment:
This extra point we are adding is not real. We should not print a timestamp for it. It is confusing for the user because we are printing the last timestamp twice. The better approach is not to print any timestamp. Leaving it empty is a signal that this last part is just for visualization. We can add that to the jupyter examples as well.
```suggestion
        "Date": "",
```

Reply:
There are two situations we need to consider

1. Mode of Last TS is not equal to prev TS: Having the Date as an empty string will help avoid printing the same date twice.
2. Otherwise - The date of the last row will not be printed.
![image](https://user-images.githubusercontent.com/5328735/159275953-693c56c1-8f72-45a5-86c7-d344b963fc9d.png)